storybookjs/storybook
# Sandboxes: Fix up stories entry to allow HMR

Issue: watchpack doesn't cross symlink boundaries, so changes inside symlinks weren't getting detected.

## What I did

- Set up a entry & watch for each link in `template/stories` rather than one for all
- Use a better glob to match `.mdx`
- This actually helped resolved some issues with wonky titles for a few files.

Note the issue still remains for things linked inside `stories` (ie. renderer/framework stories). We could do the same for those but we would have to mess a bit with the original glob to stop it from matching the renderer's stories twice. Should we do that @shilman?

## How to test

- Check we haven't lost any stories in Chromatic
- Run a sandbox, add a story in `lib/store/template/stories/basic.stories.ts`
